### PR TITLE
ADD: Add scan_name to nexrad datatree attributes

### PR DESCRIPTION
**Pull request by aladinor**
- [ ] Closes #xxxx
- [x] Tests added
- [x] Changes are documented in `history.md`

### CODE REVIEW

**Comment by mgrover1** [comment]
@aladinor - I think adding a test would be good here - other than that, this looks great! Thanks for the quick PR 👍

**Comment by aladinor** [comment]
working on it!

**Review by mgrover1 — APPROVED** [review]
This looks great! thanks for the contribution here @aladinor

File: docs/history.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # History
 
+## 0.8.1 (Unrealeased)
```
Comment (mgrover1) [inline docs/history.md:3]:
```suggestion
## 0.8.1 (Unreleased)
```

**Comment by aladinor** [comment]
I am finishing test for other backends.

**Comment by mgrover1** [comment]
Sounds good @aladinor - I can give it another review when you are ready!

**Review by mgrover1 — APPROVED** [review]

**Pull request merged**